### PR TITLE
Metainfo fixes. They are now usable!

### PR DESCRIPTION
**Pull request by bear1ake**
Makes metainfo usable and available for everyone, including ghosts.

### CODE REVIEW

**Comment by SconesC** [comment]
what is this and why

**Comment by Incoming5643** [comment]
It seems to be the ever elusive "OOC notes" that people rarely see on the preference screen sometimes. I don't think it's intentionally enabled on tg servers. I think it's supposed to be something akin to "character descriptions" or something? :confused: 

If I'm reading this code right these changes will make it that you can see your OWN ooc notes, but no one elses (which kind of makes the feature pointless).

Where as the original version of this attempted to allow you to see OTHERS ooc notes, but was unusable by ghosts.

**Comment by Alek2ander** [comment]
Can this be removed from the verbs list if it's not usable (disabled in config) to reduce clutter?

**Comment by SconesC** [comment]
for what purpose

also please update the desc for people who have no idea what the ambiguous 'metainfo' is

**Comment by Thunder12345** [comment]
Wasn't this introduced with the ERP verbs?

**Comment by bear1ake** [comment]
@Incoming5643, i tested it in local server. It works.

**Comment by Cheridan** [comment]
Being attached to the client is really really bad. It needs to be attached to the mob instead, so things like changelings turning into other people or admins possessing things aren't messed up.

**Comment by bear1ake** [comment]
@Cheridan, it's info about player, not mob. You want mess up player's info?

**Comment by Cheridan** [comment]
Uh, then what's the point of this then? I'm lost.

**Comment by SconesC** [comment]
i still dont understand what the fuck this is

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -478,18 +478,18 @@ Sorry Giacom. Please don't be mad :(
...
-/mob/living/proc/Examine_OOC()
+/mob/verb/Examine_OOC()
 	set name = "Examine Meta-Info (OOC)"
 	set category = "OOC"
 	set src in view()
 
 	if(config.allow_Metadata)
 		if(client)
-			src << "[src]'s Metainfo:<br>[client.prefs.metadata]"
+			usr << "[src]'s Metainfo:<br>[client.prefs.metadata]"
```
Comment (CorruptComputer) [inline code/modules/mob/living/living.dm]: I think its your client preferences.

**Comment by palpatine213** [comment]
Make this only work for ghosts, otherwise this is a terrible idea.
If you let living players see this, they will use this to meta so much, disguises will be completely useless if security sees your meta info and decides to stop you for a search because you "looked suspicious".

**Comment by Cheridan** [comment]
This seems dead and nobody seems to even know what it's purpose is.

**Pull request closed without merging**